src/Components/Admin/admin.js

```diff
@@ -255,9 +256,13 @@ export default function Admin() {
                     variant="contained"
                     color="primary"
                     className={classes.submit}
-                    onClick={() => {
+                    onClick={async () => {
                         history.push("/login");
                         auth.signOut();
```

Comment:
I originally tried to get rid of the auth.signOut here, but I think it is needed since it will appropriately update the userContext which our routing is based off of.

A future todo, which I've already created, is to move off userContext and instead look at the existence of the xsrf / dcc_auth tokens as an indicator if a user is logged in or not.